Packs/TIM_Processing/IncidentTypes/incidenttype-Review_Indicators_Manually_For_Allowlisting.json

```diff
@@ -1,15 +1,15 @@
 {
-	"id": "Review Indicators Manually For Whitelisting",
+	"id": "Review Indicators Manually For Allowlisting",
```

Comment:
@ali-sawyer 
Changing incident type id and name will affect all other places using this object.
For example, in the playbook "TIM - Process AWS indicators" (FeedAWS pack) - there's a task named "Create Process Indicators Manually incident" which creates a new incident from that type. There are others as well.
If we change the id+name of this incident type, we should iterate through all of the other places which uses this incident type and change them as well to use the new name/id so they won't break.